content/coop.html

```diff
@@ -274,7 +274,7 @@ <h3> Favourite Co-op Experiences </h3>
   <div class="vb"></div>
   <p>"Too many to pick. A funny one is once someone leaked [company X's] internal uber/lyft codes and we were taking unlimited $500 Uber Luxe XLs across California."</p>
```

Comment:
```suggestion
  <p>"Too many to pick. A funny one is once someone leaked [company X's] internal [U]ber/[L]yft codes and we were taking unlimited $500 Uber Luxe XLs across California."</p>
```

(sorry; hijacking again to fix something orz)